code/modules/shuttle/arrivals.dm

```diff
@@ -143,7 +143,8 @@
 		force_depart = FALSE
 	. = ..()
 	if(!. && !docked && !damaged)
-		console.say("Welcome to your new life, employees!")
+		if(console)
+			console.say("Welcome to your new life, employees!")

```

Comment:
Need to track the previous console's name via the persistence SS so you eventually get "Ghost of Ghost of Ghost of Ghost of Deep Fried Console"